masakari-controller/controller/masakari_util.py

```diff
@@ -53,6 +51,26 @@
 import db.api as dbapi
 from db.models import NotificationList, VmList, ReserveList
 
+from oslo_log import log as logging
+
+LOG = logging.getLogger(__name__)
+
+class LogProcessBeginAndEnd(object):
+    def output_log(self, func):
+        def _output_log(*args, **kwargs):
+            start_msg = ("BEGIN %s: parameters is %s, %s") % (func, args, kwargs)
+            LOG.debug(str(threading.current_thread()) + " " + start_msg)
+
```

Comment:
Use threading.current_thread().getName() instead of current_thread(). It shows thread name and linux time when the thread is generated.

Additionally, if possible, when creating a new Thread(), define thread name with notification id or any id/name related to db row. I think it make us easy to debug.